src/backend/aspen/database/models/usergroup.py

```diff
@@ -55,6 +55,8 @@ class Group(idbase, DictMixin):  # type: ignore
         server_default=text("'{}'::jsonb"),
     )
 
+    members = relationship("User", back_populates="group")  # type: ignore
```

Comment:
Not totally sure why these mappings are updated -- just a heads-up that user.group_id is eventually going to go away.

Reply:
😵‍💫